raceSystem/webUI.py

```diff
@@ -0,0 +1,194 @@
+from flask import Flask, render_template, request, jsonify
+from raceSystem.userInterface import UserInterface
+
+
+class WebIU(UserInterface):
+    def __init__(self) -> None:
+        self._app = Flask(__name__)
+        self._startClientConnected = False
+        self._finishClientConnected = False
+        self._raceStatus = ''
+        self._raceResult = ['', '']
+
+    def run(self):
+        """Start the user interface"""
+        self.setupFlaskRouting()
+        self._app.run(debug=True, use_reloader=False)
```

Comment:
We can do a trick here just like we do in server.py to start flask on localhost or on the ip adress of the rpi router that's used by the system.
from raceSystem.RPiTrafficLight import testMode
if testMode:
           self._app.run(debug=True, use_reloader=False)
else:
           self._app.run(host= '192.168.1.173', port=9000, debug=False, use_reloader=False)

Reply:
Adjusted as requested.

Note: I like to run the Flaks app in both option in debug mode. The debug mode means that: if an error occurs with the webserver (illegal text input, page not found, maybe a firewall block, etc....) Flask returns a full report where en how the error occurs. So, when they find a bug during a field test they are able to take a screenshot and we can debug it afterwards. For clarity: when everything works fine there will be no difference with the debugger on or off.